src/test/app/Remit_test.cpp

```diff
@@ -2102,9 +2102,8 @@ struct Remit_test : public beast::unit_test::suite
             std::string result;
             TER code;
         };
-        std::array<TestRateData, 10> testCases = {{
+        std::array<TestRateData, 9> testCases = {{
```

Comment:
Per Conan branch
https://github.com/Xahau/xahaud/pull/432/files#diff-82eae0720aa190ee8fccab6d32c93d05d702038c7ed1b18c6edf820432a9d52c

Reply:
Prob same here tbh, a single PR to fix it. I should have done the same in conan but that is also a WIP.

Reply:
https://github.com/Xahau/xahaud/pull/452